main.js

```diff
@@ -7,6 +7,33 @@ const baseOil = 1; // TS of oil
 const baseArea = Math.PI * Math.pow(28 / 2, 2); // area of a 28 cm pizza
 const baseDough = 180; // grams of dough for a 28 cm pizza
 
+const sliceStateModule = {
```

Comment:
if we go stateful anyway, should we make this a class an instantiate it once in `init()`? We could keep the HTML element references then